geowebcache/azureblob/pom.xml

```diff
@@ -25,8 +25,8 @@
       <version>11.0.0</version>
       <exclusions>
         <exclusion>
-          <groupId>org.slf4j</groupId>
           <artifactId>slf4j-api</artifactId>
+          <groupId>org.slf4j</groupId>
```

Comment:
Seeing a group id after the artifact id seems strange. Is sortpom not complaining?

Reply:
it probably did, I am having trouble running these things locally.

Reply:
Okay you are correct dependency management overrides the transitive dependency version. Kind of scary, but okay...